mule-migration-tool-library/src/main/java/com/mulesoft/tools/migration/library/tools/mel/nocompatibility/PropertiesNoCompatibilityResolver.java

```diff
@@ -137,9 +141,9 @@ private String replaceAllOccurencesOfProperty(String content, Matcher outerMatch
           if (propertyTranslation == null) {
             propertyTranslation = tryImplicitTranslation(propertyToTranslate, flowComponent);
             if (propertyTranslation == null) {
-              report.report("nocompatibility.unsupportedproperty", flowComponent.getXmlElement().getParentElement(),
-                            flowComponent.getXmlElement().getParentElement(), flowComponent.getXmlElement().getName());
-              failedCompleteTranslation = true;
+              propertyTranslation = fallbackTranslation(propertyToTranslate);
+              logger.info("Property '{}' not found in context, using fallback translation '{}' ", propertyToTranslate,
```

Comment:
I think I put another comment asking for a log and now I see this here, so disregard the other one